specs/spec_reader.go

```diff
@@ -12,11 +13,31 @@ type SpecReader struct {
 	Destinations map[string]*Destination
 }
 
+var fileRegex = regexp.MustCompile(`\$\{file:(.+)\}`)
```

Comment:
```suggestion
var fileRegex = regexp.MustCompile(`\$\{file:([^}]+)\}`)
```

I think this regex should also handle the case where multiple variable replacements occur in the same line. 

E.g. before:

```
${file:abc}-${file:def}
```

would greedily match `abc}-${file:def` as the filename, instead of two separate matches `abc` and `def`. (Assuming this is a valid)

We may want to add a test case where two variable replacements happen as well (maybe one file replacement and one normal replacement)

Reply:
Done.

Reply:
Why not do `var fileRegex = regexp.MustCompile(`\$\{file:(.+?)\}`)` for an ungreedy match?